examples/fabric-bridge-app/linux/include/RpcClient.h

```diff
@@ -41,4 +41,5 @@ CHIP_ERROR InitRpcClient(uint16_t rpcServerPort);
  * - CHIP_ERROR_BUSY: Another commissioning window is currently in progress.
  * - CHIP_ERROR_INTERNAL: An internal error occurred.
  */
-CHIP_ERROR OpenCommissioningWindow(chip::NodeId nodeId);
+CHIP_ERROR OpenCommissioningWindow(chip::NodeId nodeId, uint16_t commissioningTimeout, uint16_t discriminator, uint32_t iterations,
```

Comment:
This is a method with 6 arguments. I say after 3-4 arguments we should re-think how they are passed in.

Reply:
Introduced param builder for example in https://github.com/project-chip/connectedhomeip/pull/34209/commits/d29e4fc464816478013110707686ad8d75a5d806